stripe/_http_client.py

```diff
@@ -282,9 +281,6 @@ def _record_request_metrics(self, response, request_start, usage):
                 request_id, request_duration_ms, usage=usage
             )
 
-
-class HTTPClient(HTTPClientBase):
```

Comment:
What's the benefit for getting rid of `HTTPClientBase` here? It looks like this inheritance is also [in the master branch](https://github.com/stripe/stripe-python/blob/aa7d8bf305eb6262919b5237cfebea6491de1b8c/stripe/_http_client.py#L109) so maybe we should consider removing it in master too to err on the safe side of merge conflicts.

Reply:
Oh, it's just unused now. I think it is safe to remove in master as isn't directly exposed publicly: opened the PR here: https://github.com/stripe/stripe-python/pull/1243 I can merge that prior to this one.